wot-wg-2023-draft.html

```diff
@@ -380,26 +380,24 @@ <h3>
               This will be a revision of the current <i>Web of Things (WoT) Discovery</i> specification published
               at <a href="https://www.w3.org/TR/wot-discovery/">https://www.w3.org/TR/wot-discovery/</a>,
               as a Candidate Recommendation on 19 January 2023.</p>
-          </dd>
-          <dt id="binding-deliverable" class="spec">Web of Things (WoT) Binding Templates</dt>
-          <dd>
-            <p>
-              This specification defines the mechanism to bind protocols, payload and media formats, or combination of
-              the two to the Web of Things.
-              The binding mechanism will be a normative document with the following set of registry sections:
-            <ul>
-              <li>3 registry sections dedicated to stable bindings that reach at least a working draft status</li>
-              <li>3 registry sections dedicated to experimental bindings that are at least publicly available</li>
-            </ul>
-            </p>
+            </dd>
+            <dt id="binding-deliverable" class="spec">Web of Things (WoT) Binding Templates</dt>
+            <dd>
+              <p>
+                This specification defines the mechanism to bind protocols, payload and media formats in order to
+                allow integration of ecosystems into the Web of Things.
```

Comment:
Nit: I would probably have said "different IoT platforms" rather than "ecosystems".

Reply:
I find this kind of naming very difficult since sometimes these are standards (like OPCUA and ECHONET) and sometimes they are a software stack in form of a platform (like philips hue, alexa, even webthings and similar smart home solutions)

Also see https://github.com/w3c/wot-binding-templates/issues/226

Reply:
I personally am ok with ecosystems.  Platforms implies a software stack, not all ecosystems are related to standards.  So this is the best of the bad options.